docs/src/operate/c8y/c8y_service_monitoring.md

```diff
@@ -11,59 +11,62 @@ or on the `child` device can be monitored from the **Cumulocity IoT** by sending
 
 ## Send the health status of a service to health topic.
 
-The table below lists the MQTT topics to which the health status message should be sent, and the
-health status message format for both the `thin-edge` and for the `child` device.
+A health status message can be published for any service on a `status/health` channel. The health message has to contain
+at least the `status` of the service.
 
-|Device-type|Thin-edge-Health-status-mqtt-topic|Health-status-message|
-|------|------------------------|---------------------|
-|Thin-edge-device|`tedge/health/<service-name>`|`{"status":"status of the service","type":"type of service"}`|
-|Child-device|`tedge/health/<child-device-id>/<service-name>`|`{"status":"status of the service","type":"type of service"}`|
+**TODO: Link to the Health Status Message reference!**
 
 :::note
 The `status` here can be `up or down` or any other string. For example, `unknown`.
 :::
 
-For example, to monitor the health status of a `tedge-mapper-c8y` service that is running on a `thin-edge.io` device
-one has to send the below message.
+For example, to update the health status of `device/my-device/service/my-test-service` service, one has to send the
+following message:
 
-```sh te2mqtt
-tedge mqtt pub tedge/health/tedge-mapper-c8y '{"status":"up","type":"thin-edge.io"}' -q 2 -r
-```
-
-The above message says that the `tedge-mapper-c8y` is `up` and the `type` of the service is `thin-edge.io`.
-
-
-To monitor the health of a `docker` service that is running on an `external-sensor` child device,
-
-```sh te2mqtt
-tedge mqtt pub tedge/health/external-sensor/docker '{"status":"up","type":"systemd"}' -q 2 -r
+```sh te2mqtt formats=v1
+tedge mqtt pub te/device/my-device/service/my-test-service/status/health '{"status":"up"}' -q 2 -r
 ```
 
 :::note
 The health status message has to be sent as a `retain` message.
 :::
 
-When an empty health status message is sent, e.g. `{}` or `''`, the `status` will be replaced with `unknown` and the `type` will be replaced with default value `service`.
+When an empty health status message is sent, e.g. `{}` or `''`, the `status` will be replaced with `unknown`.
```

Comment:
what about the `default` service type?

Reply:
`type` parameter was removed from the health message, and now is only found in the registration message.